.gitignore

```
@@ -71,3 +71,6 @@ crashlytics-build.properties
 # Temporary auto-generated Android Assets
 /[Aa]ssets/[Ss]treamingAssets/aa.meta
 /[Aa]ssets/[Ss]treamingAssets/aa/*
+
+.sln
+.slnx
```

```suggestion

*.slnx
```

Probably my only suggestion since `*.sln` already exists.